components/Workout/WorkoutHeader.js

```diff
@@ -12,12 +57,24 @@ const HeaderContainer = styled.section`
   width: 100%;
   background-color: #fcfbff;
   height: 4.5rem;
-  position: absolute;
+  position: fixed;
   box-shadow: 0px 0px 10px #463f3a;
   top: 0;
   display: flex;
   align-items: center;
-  justify-content: space-around;
+  justify-content: center;
+`;
+
+const QuitIcon = styled(Icon)`
+  position: absolute;
+  left: 20px;
+  top: 20px;
+`;
+
+const SaveIcon = styled(Icon)`
+  position: absolute;
+  right: 20px;
+  top: 20px;
```

Comment:
Two components, same styling - couldn't this be a `StyledIcon` component for all icons?